NitroxServer/Serialization/World/WorldManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NitroxModel.Helper;
+
+namespace NitroxServer.Serialization.World;
+
+public static class WorldManager
+{
+    public static readonly string SavesFolderDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Nitrox", "saves");
+
+    public static readonly string[] WorldFiles = { "BaseData.json", "EntityData.json", "PlayerData.json", "Version.json", "WorldData.json" };
+
+    private static readonly List<Listing> savesCache = new();
```

Comment:
```suggestion
    private static readonly IList<Listing> savesCache = new();
```

Reply:
What's the purpose of this change? I can't sort the list based on the "LastAccessedTime" with `IList`